test/unit/reducer/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+###############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
```

Comment:
What about hip and tbb?

Reply:
I'll add HIP in soon. TBB is lumped in with the *-host.cpp because it uses the Host resource in other tests.

Reply:
I understand, but please instantiate tests for each back-end in their own source file.

Reply:
@rhornung67 I believe all of the issues (individual backend source files, HIP, removing old tests) have been addressed. Would you mind taking another look?

Reply:
It looks like all of the old reduction unit tests have been removed. If not, we can take another pass at removing them in a subsequent PR.